doc/references.bib

```diff
@@ -396,3 +396,15 @@ @article{lloyd2018quantum
   title = {Quantum Generative Adversarial Learning},
   journal = {Physical Review Letters}
 }
+
+@article{clements2016optimal,
+  doi = {10.1364/OPTICA.3.001460},
+  year  = {2016},
+  publisher = {OSA Publishing},
+  volume = {3},
+  number = {12},
+  pages = {1460--1465},
+  author = {Seth Lloyd and Christian Weedbrook},
```

Comment:
??? 
Let's not give these guys credit :)

Reply:
lol